### PR TITLE
wolfram_alpha: Check if the pods exist

### PR DESCRIPTION
**Pull request by meetmangukiya**
There are questions/strings that doesn't yield any pods at all, which raise
KeyError.

Fixes https://github.com/coala/corobo/issues/161

# Reviewers Checklist

- [ ] Appropriate logging is done.
- [ ] Appropriate error responses.
- [ ] Handle every possible exception.
- [ ] Make sure there is a docstring in the command functions. Hint: Lookout for
  `botcmd` and `re_botcmd` decorators.
- [ ] See that 100% coverage is there.
- [ ] See to it that mocking is not done where it is not necessary.

### CODE REVIEW

File: plugins/wolfram_alpha.py [hunk 1]
```diff
@@ -21,8 +21,11 @@ def wa(self, msg, arg):
...
-        for pod in res.pods:
-            if pod.title in ['Result', 'Results']:
-                for sub in pod.subpods:
-                    ans += sub.plaintext
+        try:
+            for pod in res.pods:
+                if pod.title in ['Result', 'Results']:
+                    for sub in pod.subpods:
+                        ans += sub.plaintext
+        except KeyError:  # pragma: no cover, for logging
```
Comment (Udayan12167) [inline plugins/wolfram_alpha.py]: Can't you create a mock logger or a sample logger and check coverage?

Comment (meetmangukiya) [inline plugins/wolfram_alpha.py]: Didn't think it was necessary, but added the tests now :+1:

**Comment by Udayan12167** [comment]
ack https://github.com/coala/corobo/pull/162/commits/7c3ac8028dc918bad59825591c514025ee84237c

**Comment by meetmangukiya** [comment]
@rultor merge

**Pull request merged**